eogrow/pipelines/byoc.py

```diff
@@ -27,25 +28,40 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+TIMESTAMP_FORMAT = "%Y-%m-%dT%H-%M-%S"  # TODO: import from .export_maps once MRs are synced
```

Comment:
Why not just change the source/rebase branch to the other branch, this way the changes would automatically be included.

Reply:
because this was done before that branch was merged and it would break the tests
cleaned now